MetaMask/metamask-extension
# fix: update to transaction-controller 28.1.1 to fix NFT mint simulation

## **Description**
Update transaction-controller core package to bring in this fix to NFT mint simulation: https://github.com/MetaMask/core/pull/4217

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24306?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/24251

## **Manual testing steps**

1. Visit https://www.fxhash.xyz/generative/slug/g-l-y-p-h 
2. Try minting the nft
3. Simulation should work and show the nft in the "You receive" section.

## **Screenshots/Recordings**



### **Before**

(Simulation Failed message)

### **After**

![image](https://github.com/MetaMask/metamask-extension/assets/507015/ded0173a-088c-4f12-b3b6-09b823db7756)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
